### ISSUE TITLE
Concept: Easy-to-use UI (See video!)

### ISSUE THREAD

**Issue by rpertusio** [NatroTeam/NatroMacro#276]
### Have you searched the existing suggestions?

Yes

### What is your suggestion?

**Existing problems**
- Natro is great! But AutoHotKey default dialogs are plain
- Some options are hard to understand, especially for new users, such as Sprinkler/pattern dropdowns

**I'm proposing:**
- A migration to a Bootstrap interface (via AutoHotKey + Neutron)
- Make certain actions easy to understand, such as:  Drag & Drop sprinkler, Field pictures instead of names, color-coded buttons, etc.

**I have put together a sample video of what could be possible:**
https://github.com/NatroTeam/NatroMacro/assets/2339691/f7105da7-a722-4297-a930-b09bbb7f1c9b
(This is just a semi-working UI sample. It runs in AHK+Neutron, but is just an HTML+Javascript that doesn't actually drive Natro code.)

I am somewhat comfortable coding it myself, although it would require a few core changes to Natro (such as accepting more than 3 fields, new layout for entire UI).   I'm a 'weekend coding warrior', so it's not going to be done fast.  Expect bugs.  But, the potential is great!

This suggestion is to gauge Natro developers' interest in such a fork/project for a v2.0 of Natro.

### How will this improve Natro Macro?

- Easier adoption of Natro
- Less questions in Discord/GitHub
- Makes it look polished and professional

### What is your Discord username?

CPx4

**Comment by rpertusio** [comment]
@Noobyguy775 this would take a lot of time to do. But if the Natro team is up for it, I'm possibly up for tackling it.  Any thoughts? Am I stepping on any work you're already doing?  (Would there be any hard feelings if I revamp the UI?)

Lastly, is there a Developers corner in Discord I could bounce ideas off of?

**Comment by Noobyguy775** [comment]
> @Noobyguy775 this would take a lot of time to do. But if the Natro team is up for it, I'm possibly up for tackling it. Any thoughts? Am I stepping on any work you're already doing? (Would there be any hard feelings if I revamp the UI?)
> 
> Lastly, is there a Developers corner in Discord I could bounce ideas off of?

No, you're not interrupting any work currently being done (at least on my part). The Natro UI is very memorable and has a "unique" style, which I've only seen AHK GUI's have. However, the amount of customization possible with the UI you showcased is amazing, like the sprinkler part, and just the amount of customization compared to the `GUI` function in AHK.

About a Dev corner on Discord, you should dm `xspx` on Discord (@zspz)
